lib/corsac_jwt.dart

```diff
@@ -83,6 +83,9 @@ class JWT {
 
   Map<String, String> _headers;
   Map<String, dynamic> _claims;
+  
+  // Allows access to the full claims Map
+  Map<String, dynamic> get claims => _claims;
```

Comment:
This exposes internal Map for modification, which would be really nice to avoid. How about returning a copy of it each time someone asks for claims?

Reply:
Good call!